lms/djangoapps/edraak_certificates/edraakcertificate.py

```diff
@@ -0,0 +1,620 @@
+
```

Comment:
@shadinaif please rename back to `generator.py` to preserve the old git history.